src/main/daml/Daml/Finance/Lifecycle/SettlementRule.daml

```diff
@@ -85,7 +85,7 @@ template Rule
         newInstrumentHoldingCids <- fmap sequence . mapA updateInstrumentHolding $ zip holdings holdingCids
 
         -- Generate settlement instructions for other instruments
-        (settleableCid, instructionCids) <- exercise instructableCid Instructable.Instruct with settler; id = effectView.id; steps
+        (settleableCid, instructionCids) <- exercise instructableCid Instructable.Instruct with instructors = fromList [custodian, owner] ;settler; id = effectView.id; steps
```

Comment:
This is the biggest change. In the case of a derivative, by default we are making the custodian and owner sign the batch.

This would previously be signed by the `requestor` (typically being one of the owner or the custodian).

An alternative would be to have the settler sign the batch + instructions, but we would need to first collect their authorization.